UnrealPlugin/LinuxBuildsUE5.md

```diff
@@ -0,0 +1,99 @@
+# Setting up a Linux Dedicated Server on PlayFab
+
+The purpose of this guide is to describe all the steps needed to build and verify a Linux dedicated server. This was tested on both Unreal Engine 5 early access and Unreal Engine 4.26 but the following steps will be specific to Unreal Engine 5, early access.
+
+More Linux deployment information can be found [here](https://github.com/PlayFab/MpsAgent/blob/main/LocalMultiplayerAgent/MultiplayerSettingsLinuxContainersOnWindowsSample.json)
+
+## Requirements
+* Install [Unreal Linux prerequisites](https://docs.unrealengine.com/4.27/SharingAndReleasing/Linux/GettingStarted/)
+* Install [Docker for Windows](https://docs.docker.com/desktop/windows/install/)
+
+## Instructions
+
+* From **Unreal Linux prerequisites**, download the toolchain for your installed Unreal Engine and run the executable. Please note, like many parts of these guides, this will work only if your Unreal Engine built from source.
+    * After the installation is complete, examine your Windows environment variables and confirm the following system variable exists: **LINUX_MULTIARCH_ROOT**.
+* In your Unreal Engine source directory:
+    * Rerun: ```Setup.bat```
+    * Rerun: ```GenerateProjectFiles.bat```
+* Return to your own project:
+    * Right-click your .uproject file, and **Generate Visual Studio project files**
+    * Open the .sln file in Visual Studio
+    * Set the configuration to **Development Editor**
+    * Build and run the Unreal Editor
+* At this point, you should be able to package your project for Linux server in a directory of your choice.
+    * For Unreal Engine 5, in the Editor, click packaging on the top right of the screen and the Linux option should now be available to be selected.
+    * Confirm your configurations for Server & Shipping, choose Linux to build, and then select the output directory.
+
+## Create Docker Image of the Linux Server
+
+In this section, we'll set up a Docker image that contains your Linux Server
+
+* Now that you have created a Linux server, navigate to that directory (Windows Explorer, or terminal of your choice)
+* Create a new file named: ```dockerfile```
+* Edit the file with your editor of choice (notepad, vim, emacs etc.)
+* Copy - paste the content presented below. Make sure to replace [YourGameNameServer].sh with the name of your own linux server executable (in the same directory)
+
+```Docker
+FROM ubuntu:18.04
+
+# Unreal refuses to run as root user, so we must create a user to run as
+# Docker uses root by default
+RUN useradd --system ue
+USER ue
+
+EXPOSE 7777/udp
+
+WORKDIR /server
+
+COPY --chown=ue:ue . /server
+CMD ./YourGameNameServer.sh
```

Comment:
@nassosterz-ms I think this line should be removed since the other one below will work. Also, can we add a comment on why we're doing that?

Reply:
Sure

Reply:
Good catch. I also added: USER root that was missing. That happenned because I copied pasted the dockerfile from here:
https://github.com/narthur157/playfab-gsdk-ue4

But the correct implementation that is working and currently tested is here: 
https://github.com/PlayFab/gsdk/blob/master/UnrealPlugin/ThirdPersonMPCloudDeploy.md

Finally, now the logs are present on GameLogs.